lib/rucio/transfertool/transfertool.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2018 CERN for the benefit of the ATLAS collaboration.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Authors:
+# - Martin Barisits <martin.barisits@cern.ch>, 2018
+
+from abc import ABCMeta, abstractmethod
+
+
+class Transfertool():
```

Comment:
class Transfertool(Object):

provocative question: do we need a class  ? to keep the host as a class member but otherwise why?

Reply:
Will add object.
I don't super like it as an object as well, but there is no other good way in python to enforce interfaces really.